Source/encrypt.cpp

```diff
@@ -70,7 +70,7 @@ void InitHash()
 			seed = (125 * seed + 3) % 0x2AAAAB;
 			ch = (seed & 0xFFFF);
 			seed = (125 * seed + 3) % 0x2AAAAB;
-			hashtable[j][i] = ch << 16 | (seed & 0xFFFF);
+			hashtable[j][i] = ch * 65536 | (seed & 0xFFFF);

```

Comment:
lets not touch hashing/encryption where bitshifts are commonly done on purpose

Reply:
Yup, encryption code should be kept cryptic :D (just kidding)